app/views/posts/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  </head>
+  <body>
+    <h1><%= @message %></h1>
+  </body>
+</html> 
```

Comment:
- [ ] `<!DOCTYPE html>` is already defined in `#app/views/layouts/application.html.erb` so you don't have to redefine in each view, only  `<h1>Posts show page</h1>` would be enough.

Reply:
@ShoiraTa. Thanks for the explanation. I can't remember reading that concept in the documentation. I will make the requested change.